README.md

```diff
@@ -1,2 +1,64 @@
-# tw-academy-s9
-Homework repository
+# Sample Style Guide
+Technical Writing Academy - May 2021
+
```

Comment:
A list of all the sections would be a good opportunity to include cross-references in this markdown showcase.

Reply:
Ok! Done!